sig-cli/annual-report-2024.md

```diff
@@ -30,17 +30,12 @@
       Please raise an issue in kubernetes/community, if the KEP metadata is correct but the generated list is incorrect.
 -->
 
-<!-- 
-  - Alpha
-    - [3104 - Introduce kuberc](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/3104-introduce-kuberc) - v1.32
```

Comment:
KEP was merged, and [PR](https://github.com/kubernetes/kubernetes/pull/125230) was pending(will be target to v1.33.)